packages/iobio-react-components/src/utils/stats.mts

```diff
@@ -19,18 +19,19 @@
  *
  */
 
-import fs from 'node:fs';
-
 import { DataBroker } from 'iobio-charts/data_broker.js';
-
+import fs from 'node:fs';
 import { calculateMeanCoverage, getBamStatistics } from './index.ts';
 
-// TODO: Add env support
-const fileUrl = process.argv[2];
+if (!process.argv[2])
+	throw new Error('Alignment URL is required to generate statistics \nusage: pnpm run stats ${url}');
```

Comment:
Ideally will further improve UX / option handling with Commander in https://github.com/overture-stack/iobio-components/issues/73

https://www.npmjs.com/package/commander

Reply:
I see very similar output on my end, both `throw` and `throw new` contain stack traces